s-port_flashing.md

```diff
@@ -1,2 +1,108 @@
-# S-PORT flashing
-TO-DO
+# S.Port Firmware Flashing
+
+## File Preparation
+
+- Download the firmware
+
+  ![](images/sport-flash-001-download-firmware.png)
+  
+- Place the firmware on the SDCARD in the FIRMWARE folder
+  
+  ![](images/sport-flash-104-sd-hc-firmware.bmp)
+
+- Ensure that the file name is 33 characters or less including the dot and extension
+
+  ![](images/sport-flash-106-sd-hc-firmware-file-selected.bmp)
+  
+## Hardware Preparation
+
+### All transmitters except for Taranis Q X7 shipped with bottom S.Port pins
+
+- The external module bay’s +ve pin has the same voltage as the transmitter’s battery
+
+  Check that the device being flashed is rated for the transmitter’s battery voltage
```

Comment:
Add a paragraph: This is true for X9D+ standard batteries (6 cell Nimah) and most Sport devices . For X12S and X9E this is not true but only slightly outside the range and reportely works. But try at your own risk.